src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -944,18 +964,66 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         self ! CallFailed(collector, None, e.getMessage)
       case Success(outputs) =>
         logger.info(s"Collection complete for Scattered Call ${collector.tag}.")
-        self ! CallCompleted(collector, outputs, 0)
+        self ! CallCompleted(collector, outputs, 0, "")
     }
 
     Success(ExecutionStartResult(Set(StartEntry(collector, ExecutionStatus.Starting))))
   }
 
+  private def sendStartMessage(callKey: CallKey, callInputs: Map[String, WdlValue]) = {
+    def registerAbortFunction(abortFunction: AbortFunction): Unit = {}
+    val backendCall = backend.bindCall(workflow, callKey, callInputs, AbortRegistrationFunction(registerAbortFunction))
+
+    /* Tries to use the cached Execution to send a UseCachedCall message.  If anything fails, send an InitialStartCall message */
+    def loadCachedCallOrInitiateCall(cachedDescriptor: Try[WorkflowDescriptor], cachedExecution: Execution) = cachedDescriptor match {
+        case Success(descriptor) =>
+          descriptor.namespace.resolve(cachedExecution.callFqn) match {
+            case Some(c: Call) =>
+              val index = if (cachedExecution.index == -1) None else Option(cachedExecution.index)
```

Comment:
You can use `val index = cachedExecution.index.toIndex` here, there is an implicit class in engine.package to convert indexes from/to Int/Option